docs/user-guide/cli-using-initializing-team-configuration.md

```diff
@@ -48,3 +48,25 @@ The following steps describe how to interactively initialize team profile config
 
 **Important:** After the configuration files are in place (either via the zowe config init command or by manually creating the files), the now-deprecated zowe profiles commands will no longer function. Zowe CLI will return errors if you attempt to use deprecated profile commands.
 
+## Connecting profiles to API Mediation Layer
+
+To configure your profile to connect to API ML, issue the following command to set up your initial `zowe.config.json` file:
+
+```
+zowe-config-auto-init
+```
+
+After you issue the command, Zowe CLI prompts you to specify the following information:
+
+- The host name and port to your API ML instance
+- Your username and password
+
+If you are using certificates to authenticate, you can specify the details for the certificates by issuing the following commands:
+
+```
+zowe auth login apiml --cert-file
```

Comment:
This will be `zowe config auto-init --cert-file "/path/to/cert/file" --cert-key-file "/path/to/cert/key/file"`. However, this is on the near-term backlog and hasn't been implemented just yet. @awharn will likely be implementing it.

Reply:
ok,  will implement this change at a later date.  i'll ping andrew to remind me